compose/local/django/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax = docker/dockerfile:experimental
-FROM python:3.8
+FROM public.ecr.aws/alexaprizesharedresources/python:3.7-slim-buster
```

Comment:
Can you use a non slim buster for now, and once you get a successful deploy, switch to slim buster?